src/components/apps/listing/TableView.js

```diff
@@ -0,0 +1,240 @@
+                <EnhancedTableHead
+                    selectable={true}
+                    numSelected={selected.length}
+                    rowsInPage={listing?.apps.length}
+                    order={order}
+                    orderBy={orderBy}
+                    baseId="appTableView"
+                    columnData={columnData}
+                    onRequestSort={handleRequestSort}
+                    onSelectAllClick={handleSelectAllClick}
+                />
+                {loading && (
+                    <TableBody>
+                        <TableLoading numColumns={6} numRows={25} />
+                    </TableBody>
+                )}
+                {!loading && (
+                    <TableBody>
+                        {(!apps || apps.length === 0) && (
+                            <EmptyTable
+                                message={getMessage("noApps")}
+                                numColumns={columnData.length}
+                            />
+                        )}
+                        {error && (
+                            <EmptyTable
+                                message={error.toString()}
+                                numColumns={columnData.length}
+                            />
+                        )}
+                        {apps &&
+                            apps.length > 0 &&
+                            apps.map((app, index) => {
+                                const {
+                                    average: averageRating,
+                                    user: userRating,
+                                    total: totalRating,
+                                } = app.rating;
+                                const appId = app.id;
+                                const appName = app.name;
+                                const isSelected =
+                                    selected.indexOf(appId) !== -1;
+                                const rowId = build(baseId, appId);
+                                return (
+                                    <TableRow
+                                        role="checkbox"
+                                        tabIndex={-1}
+                                        hover
+                                        selected={isSelected}
+                                        aria-checked={isSelected}
+                                        key={app.id}
+                                        id={rowId}
+                                        onClick={(event) =>
+                                            handleClick(event, appId, index)
+                                        }
+                                    >
+                                        <TableCell padding="checkbox">
+                                            <DECheckbox
+                                                checked={isSelected}
+                                                tabIndex={0}
```

Comment:
I haven't tried this, but if this tabIndex is 0, even though the `TableRow` tabIndex is -1, can the user still reach this by using the keyboard?

Reply:
Don't know..I think the tabIndex was a copy-pasta..I will double check.